Formula/gz-math8.rb

```diff
@@ -52,6 +53,15 @@ def install
       add_executable(test_cmake test.cpp)
       target_link_libraries(test_cmake gz-math8::gz-math8)
     EOS
+    system "pkg-config", "gz-math8"
+    cflags = `pkg-config --cflags gz-math8`.split
+    system ENV.cc, "test.cpp",
+                   *cflags,
+                   "-L#{lib}",
+                   "-lgz-math8",
```

Comment:
Can we get the linker flags from `pkg-config` as well?

Reply:
sure thing: https://github.com/osrf/homebrew-simulation/pull/2724/commits/6e19949b8f19770b466b62239f6ce5ab2165572a

the sdformat14 formula is only using the `cflags` from pkg-config, but I copied the approach for using linker flags from a gz-physics formula